website/source/api/acl/roles.html.md

```diff
@@ -0,0 +1,462 @@
+---
+layout: api
+page_title: ACL Roles - HTTP API
+sidebar_current: api-acl-roles
+description: |-
+  The /acl/role endpoints manage Consul's ACL Roles.
+---
+
+-> **1.5.0+:**  The APIs are available in Consul versions 1.5.0 and later. The documentation for the legacy ACL API is [here](/api/acl/legacy.html)
+
+# ACL Role HTTP API
+
+The `/acl/role` endpoints [create](#create-a-role), [read](#read-a-role),
+[update](#update-a-role), [list](#list-roles) and [delete](#delete-a-role)  ACL roles in Consul.
+For more information about ACLs, please see the [ACL Guide](/docs/guides/acl.html).
+
+## Create a Role
+
+This endpoint creates a new ACL role.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `PUT`  | `/acl/role`                  | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes),
+[agent caching](/api/index.html#agent-caching), and
+[required ACLs](/api/index.html#acls).
```

Comment:
*Warning* This the /api/index.html page might be updated with the filtering PR. It will affect the location of these sections (expect maybe ACLS)